### PR TITLE
Add homepage masthead and profile cards

### PR DESCRIPTION
**Pull request by amitaibu**
joxi_screenshot_1472587730564

### CODE REVIEW

**Comment by amitaibu** [comment]
@snoyberg here's a start of what I had in mind, which is baed on Haskellers, but dedicated to Yesoders. Most of Haskellers' functionality isn't used because there are other/ better alternatives (e.g. discussion boards are now stackoverflow, polls is now in twitter, etc').

However, one thing that seems to be missing is a central place where we could recognize each other and know each others status (seeking job, service providers etc).

I imagine a site where users can login via GitHub, add a few words about themself, their status, maybe their "mentors" from the community, a link to their sites/ packages.

It can also showcase some "real world" sites, link to related meetups, service providers etc' - stuff that will make more people confident about the choice of going with Yesod. But of course it will happen one step at a time.

As for the theme - I just threw some Semantic UI in, to make it look modern and responsive - but obviously it can and should change if you'd think this idea is worthwhile.

**Comment by snoyberg** [comment]
Looks like a very nice homepage, and I'd agree with removing those other features.

**Comment by amitaibu** [comment]
Awesome. Would you be ok if I'll tag you once a PR is ready for review?

File: Handler/Home.hs [hunk 1]
```diff
@@ -4,46 +4,10 @@ module Handler.Home
...
-    now <- liftIO getCurrentTime
-    let minus7d = addUTCTime ((-1) * 60 * 60 * 24 * 7) now
-    let users =
-            if null allProfs
-                then []
-                else take 24 $ shuffle' allProfs len gen
-
-    let fuzzyDiffTime = humanReadableTimeDiff now
-    public <- runDB $ count [ UserBlocked ==. False]
+    -- @todo: how to add random - Must we use esqueleto with rand_?
```
Comment (amitaibu) [inline Handler/Home.hs:9]: Opened #2 as follow up.

@snoyberg if you get a chance to look at that issue, as I believe it applies to haskellers as well.

**Comment by amitaibu** [comment]
Ready for review, if anyone is around :)

**Comment by amitaibu** [comment]
I'll get this in, as we have other PRs based on it.

**Pull request merged**